services/sources/publisher_backlist_service.py

```diff
@@ -133,9 +133,9 @@ def store_pdf_manifest(self, record):
 
             if media_type == 'application/pdf':
                 record_id = record.identifiers[0].split('|')[0]
-                manifest_path = 'manifests/{}/{}.json'.format(source, record_id)
+                manifest_path = 'manifests/pubBacklist/{}/{}.json'.format(source, record_id)
```

Comment:
Could you use self.prefix here so that you don't have to update this prefix in two places?

Reply:
Sure I can do that